console/src/components/Services/Data/TableBrowseRows/EditActions.js

```diff
@@ -124,6 +127,57 @@ const editItem = (tableName, colValues) => {
   };
 };
 
+const fetchEnumOptions = () => {
+  return (dispatch, getState) => {
+    const {
+      tables: { allSchemas, currentTable, currentSchema },
+    } = getState();
+
+    const requests = enumColumnMapping(allSchemas, currentTable, currentSchema);
+
+    if (!requests) return;
+
+    const options = {
+      method: 'POST',
+      credentials: globalCookiePolicy,
+      headers: dataHeaders(getState),
+    };
+    const url = Endpoints.query;
+
+    requests.forEach(request => {
+      const req = {
```

Comment:
Please extract out the select query into a function called `getSelectQuery` in `v1QueryUtils`. For #DRY purposes.
I'm sorry because I know it seems like a bit of unnecessary pain :stuck_out_tongue: . But it'll go a loooong way when we refactor the `Data` service.